frontend/test/unit/specs/components/v-content-report-form.spec.js

```diff
@@ -84,42 +84,48 @@ describe("VContentReportForm", () => {
   })
 
   it("should render thank you note when report is sent", async () => {
-    const { getByText } = render(VContentReportForm, options)
+    const { queryByText } = render(VContentReportForm, options)
```

Comment:
We need to add similar `expect` statements to the test above, "should contain the correct contents". 
~~`getDmcaInput()`~~ `expect(getDmcaInput()).toBeInTheDocument()`

Reply:
Got it, thanks for the help!

Reply:
@obulat I ran `just lint` on the repo, but it gave me this error: `error: Recipe `lint` failed on line 92 with exit code 3`.
Is this something that needs to be addressed or should I push the changes anyway?

Reply:
My bad, turns out the error was because I didn't have `python3-distutils` installed

Reply:
Hmm, whenever I try to commit my changes I get a bunch of error messages.

Reply:
What kind of errors did you get?

Reply:
@ZeroPlayerRodent, do you get the errors reported in #2432? If so, they should be fixed in #2433. After it's merged, you can merge main into your PR, and they should disappear.

Reply:
@ZeroPlayerRodent, I've merged #2433, and then merged `main` into your PR, so now those errors should be gone.